index.d.ts

```diff
@@ -2312,19 +2312,19 @@ export const toaster: {
   /**
    * Opens a Toast with an intent of none.
    */
-  notify: (title: string, settings?: ToasterSettings) => void
+  notify: (title: string | React.ReactNode, settings?: ToasterSettings) => void
```

Comment:
We can simplify these title types to just be `React.ReactNode` - `React.ReactNode` is a catch-all type that encompasses `string` as a part of it.

Reply:
Good idea👍